### ISSUE TITLE
'below' instead of 'above' in Upper threshold

### ISSUE THREAD

**Issue by yw7** [spinalcordtoolbox/spinalcordtoolbox#3921]
https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/d17e52f90a971cb778d93d88306b102254e9ba2f/spinalcordtoolbox/scripts/sct_maths.py#L193

Hello SCT team,

Should it be `help='Upper threshold limit (zero above number).',`?

Thanks

**Comment by joshuacwnewton** [comment]
You're absolutely right. This seems to be a typo, given that `-thr` also says `"(zero below number)`":

https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/d17e52f90a971cb778d93d88306b102254e9ba2f/spinalcordtoolbox/scripts/sct_maths.py#L187
